components/Chat/Chat.tsx

```diff
@@ -1,21 +1,12 @@
-import { useChatProvider } from "@/providers/ChatProvider";
+import { ScrollTo } from "react-scroll-to";
```

Comment:
While ScrollTo is included, there’s no actual trigger for the scroll function to execute upon new messages. The Messages component should be auto-scrolled each time new messages are added.